src/htr2hpc/train/rm_models.py

```diff
@@ -0,0 +1,75 @@
+"""
+Utility script to delete models from eScriptorim API, for easily
+cleaning up models created and uploaded for testing.
+
+Requires an eScriptorium API token set as an environment variable in
+**ESCRIPTORIUM_API_TOKEN**.
+
+Takes a base url for the eScriptorium instance and a model name prefix; will
+delete all models that start with the specified model name prefix.
+
+usage:
+
+    python src/htr2hpc/train/rm_models.py https://test-htr.lib.princeton.edu/ model_prefix
+
+"""
+
+import os
+import argparse
+
+from tqdm import tqdm
+
+from htr2hpc.api_client import eScriptoriumAPIClient
+
+api_token_env_var = "ESCRIPTORIUM_API_TOKEN"
+
+
+def main():
+    try:
+        api_token = os.environ[api_token_env_var]
+    except KeyError:
+        print(
+            f"Error: eScriptorium API token must be set as environment variable {api_token_env_var}",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(description="Remove models from eScriptorium API")
+    parser.add_argument(
+        "base_url",
+        metavar="BASE_URL",
+        help="Base URL for eScriptorium instance (without /api/)",
+        type=str,
+    )
+
+    parser.add_argument(
+        "model_name",
+        help="Model name to remove",
+        type=str,
+    )
+    args = parser.parse_args()
+
+    api = eScriptoriumAPIClient(args.base_url, api_token=api_token)
+    model_list = api.model_list()
+    rm_models = []
+    # filter by match on name
+    while model_list.next:
+        rm_models.extend(
+            [
+                model
+                for model in model_list.results
+                if model.name.startswith(args.model_name)
```

Comment:
I see ways this could be overly greedy in deleting models with similarly-starting names, but as a pure dev use tool I'm not especially concerned.

Reply:
Yes, I thought about making it more configurable in terms of how model names are matched/filtered but figured I should keep it simple for now - we can always tweak or add more functionality later if we discover we need to.